test/test_library.py

```diff
@@ -59,7 +59,7 @@ def test_store_changes_database_value(self):
         self.i.store()
         new_year = (
             self.lib._connection()
-            .execute("select year from items where " 'title="the title"')
+            .execute("select year from items where title='the title'")
```

Comment:
While you're here it may be good to replace this with
```suggestion
            .execute("select year from items where title = ?", (i.title,))
```

Same applies to the rest of the updates. **the title** is taken out of context and it isn't clear whether it's looking for the the item that has just been stored or some other one

Reply:
See slightly below, this pattern is being used here for example

```python
    def test_albuminfo_for_two_items_doesnt_duplicate_row(self):
        i2 = item(self.lib)
        self.lib.get_album(self.i)
        self.lib.get_album(i2)

        c = self.lib._connection().cursor()
        c.execute("select * from albums where album=?", (self.i.album,))
        # Cursor should only return one row.
```

Just checked this test in question, and realised I made a typo. It should instead be

```python
            .execute("select year from items where title = ?", (self.i.title,))
```

The test works fine on my end.

Reply:
Updated to user passed values rather than hard coded values